src/Microsoft.VisualStudio.Threading.Analyzers/VSTHRD010VsServiceUsageAnalyzer.cs

```diff
@@ -119,6 +119,7 @@ public override void Initialize(AnalysisContext context)
         private static bool IsVisualStudioShellInteropAssembly(string assemblyName)
         {
             return assemblyName.StartsWith("Microsoft.VisualStudio.Shell.Interop", StringComparison.OrdinalIgnoreCase)
+                || assemblyName.Equals("Microsoft.VisualStudio.OLE.Interop", StringComparison.OrdinalIgnoreCase)
```

Comment:
I don't know. I suspect not. The editor has free-threaded components. So I'd rather have a precise whitelist.